src/app/legacy/components/MediaPlayer/index.stories.jsx

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import { boolean, withKnobs } from '@storybook/addon-knobs';
 import { CanonicalMediaPlayer, AmpMediaPlayer } from '.';
-import ampDecorator from '../../../../.storybook/helpers/ampDecorator';
+import ampDecorator from '../../../../../.storybook/helpers/ampDecorator';
```

Comment:
I may be wrong, but can't we import relative to the root just like this?
```suggestion
import ampDecorator from '.storybook/helpers/ampDecorator';
```

Reply:
Could alternatively alias the root, to something like `#`

Reply:
hmm I don't think absolute paths are supported without some extra configuration

Reply:
Sorry, I'm not keen on setting up a new alias for a legacy component. I actually think that aliases hide bad file organisation. A better fix would be not colocating the ampDecorator in the storybook. But again, I'm no so keen on investing any time on making legacy components better.